src/options-resolver.ts

```diff
@@ -57,7 +57,7 @@ export const OptionsResolver = async (options: any,
 
     let value: string
     if (options[key] !== undefined) {
-        value = options[key].trim()
+        value = typeof options[key] === 'string' ? options[key].trim() : options[key]
```

Comment:
there should be no need for the .trim

Reply:
It's ok, some users copy and paste inputs final whitespaces (e.g. keys, node URLs, addresses).